src/cgns_header.h

```diff
@@ -233,6 +233,8 @@ typedef struct {            /* IndexArray/Range_t Node      */
     CGNS_ENUMT(PointSetType_t) type;  /* PointList, PointRange, ...       */
     char_33 data_type;      /* type of data                         */
     cgsize_t npts;          /* number of points to define the patch */
+    cgsize_t range_min[12]; /* start of range if type is PointRange or ElementRange */
```

Comment:
The aim was to avoid to read the points every times I would need the ranges. I will correct It in the next commit with the introduction of the function :
`int cgi_ptset_range(cgns_ptset *ptset, cgsize_t *range_min, cgsize_t *range_max)`